1-js/02-first-steps/09-comparison/1-comparison-questions/task.md

```diff
@@ -14,5 +14,4 @@ undefined == null
 undefined === null
 null == "\n0\n"
 null === +"\n0\n"
-```
-
+```
```

Comment:
Please restore the blank line at the end of this file.

Reply:
Did you restore the new line at the of of this file? Because GitHub is showing that the new line is still removed in your changes.

Reply:
![image](https://user-images.githubusercontent.com/46679050/127764506-bacd1773-21f3-4bae-8834-99f4578fe45f.png)

Reply:
I'm sorry but you're still missing that new line at the end of the file. Possibly your editor is automatically removing it when you're saving the file. Please check.